targets/common

```diff
@@ -89,6 +89,11 @@ elif [ -z "$TARGETNOINSTALL" -o -n "$RESTOREHOSTBIN" ]; then
         $SIMULATE cp -fP "$src" "$CHROOT/etc/crouton/"
         [ ! -h "$src" ] && $SIMULATE chmod 644 "$CHROOT/etc/crouton/$f"
     done
+
+    # Copy all the source files to chroot
+    $SIMULATE rm -rf "$CHROOT/usr/src/crouton"
+    $SIMULATE cp -afP "${SRCDIR:-../src}" "$CHROOT/usr/src/crouton"
```

Comment:
`"${SRCDIR:-$TARGETSDIR/../src}"`?

Reply:
` "${SRCDIR:-../src}"` is consistent with what we do above in this file.
 - `${HOSTBINDIR:-../host-bin}`
 - `${CHROOTBINDIR:-../chroot-bin}`
 - `${SRCDIR:-../src}`

`$TARGETSDIR/../src` isn't absurd, but it's just not consistent...